easybuild/easyconfigs/m/Molden/Molden-7.3-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'MakeCp'
+
+name = 'Molden'
+version = '7.3'
+
+homepage = 'https://www.theochem.ru.nl/molden/'
+description = """Molden is a package for displaying Molecular Density from the
+ Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical
+ packages Mopac/Ampac"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://ftp.science.ru.nl/Molden/']
+sources = ['%(namelower)s%(version)s.tar.gz']
+patches = [
+    'Molden-7.3_fix_makefile_for_easybuild.patch',
+    'Molden-7.3_fix_arg_handling.patch',
+]
+checksums = [
+    {'molden7.3.tar.gz': '870f4fa6635229791bb09bbbd07f51456b2c90101d73564dc47ed7769b8c07a1'},
+    {'Molden-7.3_fix_makefile_for_easybuild.patch': '303dcd308dfb618088d014ea2956ec627d9f09b6547d547ac4b432be4e5b9d7c'},
+    {'Molden-7.3_fix_arg_handling.patch': '869eb33ea4cdd022402251b59386c146e7ceaf85f838f2a002fb4c393db4c1c2'},
+]
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('makedepend', '1.0.7'),
+]
+
+dependencies = [
+    ('X11', '20230603'),
+    ('libglvnd', '1.6.0'),
+    ('libGLU', '9.0.3'),
+]
+
+buildopts = 'CC="$CC" FC="$F90" FFLAGS="$FFLAGS -fallow-argument-mismatch" all'
+
+parallel = 1
+
+files_to_copy = [(['bin/%s' % x for x in ['ambfor', 'ambmd', 'gmolden', 'molden', 'surf']], 'bin'),
+                 'CopyRight', 'README', 'REGISTER']
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['ambfor', 'ambmd', 'gmolden', 'molden', 'surf']] + ['README', 'REGISTER'],
+    'dirs': [],
+}
+
```

Comment:
Is is possible to add some simple `sanity_check_commands` here, e.g. some commands like `molden -h`?